cloud_edition/flexibility_mode/docs/partner_starterkit.rst

```diff
@@ -0,0 +1,43 @@
+Partners Starterkit
+===================
+
+After the Flexibility setup, thanks you for taking notice of these followings points.
+
+A. Deploy your ssh keys
+-----------------------
+To get access to the environments, you need to follow this procedure:
+
+#. Make sure that the SSH keys you want to grant access are registered in the `Partners Portal`_
+#. Create a new cloud ticket to the support with following items :
+
+    - Name of your Akeneo Cloud Project (i.e. <my-project>)
+    - List of the SSH keys owners on the Partners Portal, and for each key the environment you want to provide access to (you may not want to give access to the production environment to everybody)
```

Comment:
To be complete we should ask a precise way to identify the file name of the key (don't know how it's described in the PAPO UI as I think I cannot see that..)  because some guys got several keys and sometimes you have first second name nd other with surname and company.
That would also avoid that they also post the whole key file as they do sometimes.